README.md

```diff
@@ -39,6 +39,11 @@ To configure the module behaviour, provide a non-empty JSON object to the `--exe
             // - besu: hyperledger/besu:merge
             "elImage": "",
 
+            // The log level that the EL client should log at, it overrides the value set in the `logLevel` property
```

Comment:
Niiiiice. For clarity, I'd make this:
```
            // The log level string that this participant's EL client should log at
            // If this is emptystring then the global `logLevel` parameter's value will be translated into a string appropriate for the client (e.g. if
            //  global `logLevel` = `info` then Geth would receive `3`, Besu would receive `INFO`, etc.)
            // If this is not emptystring, then this value will override the global `logLevel` setting to allow for fine-grained control
            //  over a specific participant's logging
```
(it's a pretty tricky thing to describe; I even struggled with it)